pages/petTypeEditPage.ts

```diff
@@ -21,11 +21,11 @@ export class PetTypeEditPage extends HelperBase{
     async updatePetType(){
         const petTypeInputField = this.page.getByRole('textbox')
 
+        await this.page.getByText('Update').click()
+
         if(await petTypeInputField.inputValue() === ''){
             await expect(this.page.locator('div', {hasText: 'Name'}).last()).toHaveText('Name is required')
         }
-        
-        await this.page.getByText('Update').click()
```

Comment:
I have mentioned it in my previous comment, but I ll add it one more time.
The concern with this method I have is that it combines two things:
1. Clicking of Update button
2. Validation of the message "Name is requires" **WHEN** the input field is empty.

By reading this test, tell me, is it clear where validation should happen if you don't know steps implementation?
![image](https://github.com/user-attachments/assets/5e683f97-41fd-4fae-96bb-2982559e3221)

Reply:
Oh sorry, I may have overlooked this.  I can definitely see how this is unclear.  I was trying to keep it simple, but it is ultimately not.  One minute and I'll have an update.